src/components/common/LogoComponent.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import cityspireLogo from '../../assets/imgs/cityspireLogoAlternative.png';
```

Comment:
Please remove the unused variables. thanks!

Reply:
Done!

Reply:
You really put an effort in creating a new logo, good job!